components/dashboard/farmer/RecentEntries.tsx

```diff
@@ -0,0 +1,54 @@
+import ErrorBanner from "@/components/error/ErrorBanner";
+import { jsonApiFetch } from "@/lib/axios";
+import { FarmerJournal } from "@/types/content";
+import { Node } from "@/types/highLevel";
+import { getCookie } from "cookies-next";
+import { useFormatter, useTranslations } from "next-intl";
+import useSWR from "swr";
+
+type Result = Pick<Node<FarmerJournal>, "created" | "fieldInitialstock" | "id">
+
+async function getRecentEntries() {
+  const uid = getCookie("uid")
+  return await jsonApiFetch<Result[]>(`node/farmer_daily_journal`, {
+    "filter[uid.meta.drupal_internal__target_id]": uid,
+    "sort": "-created",
+    "page[limit]": 8,
```

Comment:
Could we make this a constant somewhere? (Just so that if we wanted to allow organizations to change it later, we would not have to go searching for it?)

Reply:
E.g. for `"page[limit]": 8,`

Reply:
Fixed